### PR TITLE
ICDS - refator beneficiary detail and list

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @emord,

I refactor the queries for Beneficiary List and Details. Please let me know if you have any questions regarding this changes.

Regards,
Łukasz

### CODE REVIEW

File: custom/icds_reports/tests/reports/test_awc_reports.py [hunk 1]
```diff
@@ -2185,15 +2153,15 @@ def test_awc_report_beneficiary_82f33fa1_2aec_45ba_8d6c_d3ca9f50ab73(self):
                     "recorded_height": 0,
                     "fully_immunized": "No",
                     "person_name": "Name 4445",
-                    "sex": "F",
```
Comment (emord) [inline custom/icds_reports/tests/reports/test_awc_reports.py]: Why was this removed? Were these columns unused?

File: custom/icds_reports/tests/reports/test_awc_reports.py [hunk 2]
```diff
@@ -2457,22 +2425,54 @@ def test_awc_report_beneficiary_71230690_c828_4863_b2c1_f61a75aed9d7(self):
...
-                    "mother_name": "ममता बाई"
+                },
+                cls=DjangoJSONEncoder
+            )
+        )
+
+    def test_awc_report_beneficiary_ca040875_2e42_4ce4_acf7_f96695b370f1(self):
+        data = get_awc_report_beneficiary(
+            0, 10, 1, 'dob', 'a18', (2017, 5, 1), (2017, 3, 1), 'icds-cas'
+        )['data'][9]
```
Comment (emord) [inline custom/icds_reports/tests/reports/test_awc_reports.py]: Why did the ordering in the list change?

File: custom/icds_reports/tests/reports/test_awc_reports.py (hunk 1 above)
Comment (lwyszomi) [reply custom/icds_reports/tests/reports/test_awc_reports.py]: @emord This columns are no longer used in the beneficiary list report

File: custom/icds_reports/tests/reports/test_awc_reports.py (hunk 2 above)
Comment (lwyszomi) [reply custom/icds_reports/tests/reports/test_awc_reports.py]: @emord I found issue in limit/ordering so in tests the ordering back to normal

**Review by emord — APPROVED** [review]
